frontend/src/pages/login/confirmSignUpPage.tsx

```diff
@@ -0,0 +1,127 @@
+import Auth from '@aws-amplify/auth'
+import {
+  IonButton,
+  IonCard,
+  IonCardContent,
+  IonContent,
+  IonHeader,
+  IonInput,
+  IonItem,
+  IonLabel,
+  IonPage,
+  IonTitle,
+  IonToolbar,
+  useIonToast,
+} from '@ionic/react'
+import React, { useEffect, useState } from 'react'
+import { useRegister } from '../../model/registerContext'
+import './login.css'
+
+const ConfirmSignUpPage: React.FC = () => {
+  const [mail, setMail] = useState('')
+  const [confirmationCode, setConfirmationCode] = useState('')
+  const [presentToast] = useIonToast()
+
+  const registerContext = useRegister()
+
+  useEffect(() => {
+    setMail(registerContext.mail)
+  }, [registerContext.mail])
+
+  const confirmSignUp = () => {
+    Auth.confirmSignUp(mail, confirmationCode)
+      .then(result => {
+        console.log('Successfully confirmed: ', result)
+        Auth.signIn(mail, registerContext.password)
+      })
+      .catch(error => {
+        console.error('Error during confirmation: ', error)
+      })
+  }
+
+  const resendSignUp = () => {
+    if (mail) {
+      Auth.resendSignUp(mail).catch(error => {
+        presentToast(
+          `Fehler: ${
+            error?.message || 'Bitte versuchen Sie es später noch einmal.'
+          }`,
+          2000,
+        )
+      })
+      presentToast('Der Bestätigungscode wurde erneut gesendet', 2000)
+    } else {
+      presentToast('Bitte geben sie Ihre Mailadresse an', 2000)
+    }
+  }
+
+  return (
+    <IonPage>
+      <IonHeader>
+        <IonToolbar>
+          <IonTitle>Metapholio</IonTitle>
+        </IonToolbar>
+      </IonHeader>
+      <IonContent>
+        <IonHeader collapse="condense">
+          <IonToolbar>
+            <IonTitle size="large">Metapholio</IonTitle>
+          </IonToolbar>
+        </IonHeader>
+        <div className="container">
```

Comment:
I think we should avoid custom classes and CSS. Ionic overs a load of CSS utilities out of the box, e.g. `margin` 
https://ionicframework.com/docs/layout/css-utilities#element-margin
Let's use them and if it unavoidably we can use custom